### DISCUSSION TITLE
Final Application

### DISCUSSION THREAD

**Discussion by jesutobi** [Open-Science-Community-Saudi-Arabia/Open-Science-Community-in-Saudi#50] [Outreachy 2022] [answered]
just incase you are struggling with how to make your final application....listen to this twitter space recording by outreachy
https://twitter.com/outreachy/status/1584559899035385856?t=ARPR_-Xmb-kdaJqX-oPutQ&s=08

**Comment by BatoolMM** [comment, marked as answer]
Hi @jesutobi Let me know if you are still struggling or have any questions!

**Reply by jesutobi** [reply]
@BatoolMM i contributed to Mooc task and the OSCSA website , can i write a final application proposal on the both of them or i have to choose one
